test/message-generator/mock/jdc-mock-jds-receive-solution-while-processing-declared-job.json

```diff
@@ -0,0 +1,121 @@
+{
+  "version": "2",
+  "doc": [
+    "This test does",
+    "Mock JDC",
+    "Send SetupConnection and await for SetupConnection.Success",
+    "Send AllocateMiningJobToken and await for AllocateMiningJobToken.Success",
+    "Send DeclareMiningJob and await for ProvideMissingTransactions",
+    "Send SubmitSolution and xxx"
```

Comment:
Can we provide more detail here?

Reply:
oops this was left over from the drafting phase, thanks for catching that up

already updated!